### DISCUSSION TITLE
Missing "Mini-led setting doesn't appear to be re-applied after reboot" fix? g18 2024 - miniled wrong settings every boot.

### DISCUSSION THREAD

**Discussion by Ulvarin** [seerge/g-helper#2900] [General]
Hello,
Im owner of g18 2024 and every system boot the miniled start in multizone (need to click on anything related to screen  to make it ok, it can be 3x zone button to recycle to onezone or even a neutral color warmth button and it will work ok again)
(i believe its this problem here https://github.com/seerge/g-helper/issues/2802)
Im missing the option added in 181 version to force miniled settings on boot?
![image](https://github.com/user-attachments/assets/715ba112-0767-49be-b98b-86855bb453a6)

mentioned here:
https://github.com/seerge/g-helper/pull/2803
![image](https://github.com/user-attachments/assets/1dad8396-8e99-4ccd-ac61-b3ef20c0f783)

How do i enable it, i can not find a command in power user faq etc?

Thanks for your work, ghelper is amazing!

**Comment by seerge** [comment]
@Ulvarin hello, that change should work automatically, and app should just force last used MiniLed settings every time you start it or display configuration changes. So there is no extra checkbox under Extra :)

Do you have same model as person from that other issue? Can you upload a log from `%appdata%\GHelper`?

**Reply by Ulvarin** [reply]
Sure!
Yea, i have g18 2024 G834JYR 14900hx/4090/32GB
My default settings are visual mode default with neutral colors and gamut native on one zone backlight as on screenshot above.
[log.txt](https://github.com/user-attachments/files/16438877/log.txt)
[config.json](https://github.com/user-attachments/files/16438879/config.json)

**Reply by seerge** [reply]
@Ulvarin oke, i see that this feature is actually working for you

For example here on app start it has set last remembered miniled setting (single-zone)

```
31.07.2024 08:56:27: Miniled2 = 2 : OK
```
As it was stored in config `  "miniled": 2,`

In what situation it doesn’t match last used option? How did it work for you before I have introduced this fix (may be your device doesn’t have that firmware bug in a first place and doesn’t need any fixes?) i.e. in build 0.179 and below

**Reply by Ulvarin** [reply]
Ok ive created 30s unlisted youtube short with this problem, ( https://www.youtube.com/shorts/spELwvNzg3s )
Soo if i look closely the laptop starts in kinda right mode, taskbar looks right for a second and around ~14s the windows taskbar is getting darker again = multizone ??
Than i open up ghelper (around 30s) and everything is exacly as it should, after clicking "neutral" again it fixes itself,
It can by fixed by changing or clicking exacly the same option about backlight just like on the video.

It looks like it starts ok, but something is "looping" and changing it back to wrong setting again?

PS. Excuse my breathing, omg phone picked that up like im dying.

**Reply by seerge** [reply]
@Ulvarin on your video you adjust color temperature, it’s part of Visual Mode (or Game Visual in AC), it has nothing to do with multizone and backlight.

Its a software color profile, that is being set by AsusSplendid utility (that both AC and G-Helper can run)

**Reply by Ulvarin** [reply]
omg i think you are right.
After clean boot cycling trough one zone multi zone and multizone strong does not change anything its still "darkish" on One Zone.

But clicking on ANYTHING in visual mode fixes colors immidiately. Clicking Gamut native -> native = fixed.

Have i done something wrong? I've stopped all asus services in ghelper, nothing from asus is running.
What kind of option could be loading during boot that destroys colors but does not change anything in ghelper? its like some hidden option or variable  gets stuck but is not seen by software?

I've clicked trough all available colour options and honestly none of them look exacly  like the one i get after boot :O.
It is hard to compare.
The closest one and it might be it is Gamut DCIP3 - its like screen might be switching to this one after boot when it "darkens"?

**Reply by seerge** [reply]
@Ulvarin could your case be similar to this issue https://github.com/seerge/g-helper/issues/2907 ? User seem to resolve it with the driver update

**Reply by seerge** [reply]
@Ulvarin did you had a chance to see if a clean driver install "fixes" visual mode behavior on your system ?

Cause what is (most likely) happening in your case is that default gamut / color settings on your device from clean boot is by some reason is different from what `AsusSplendid.exe` (that both G-Helper and AC use to toggle visual modes and gamuts) sets as a `Default` Visual mode.

**Reply by Ulvarin** [reply]
Hey,
It does not and its been checked on 3 g18's :)
We have 3 units of g18 2024 (me and my 2 work friends)
All of them do same thing,
Laptop boots ok, and at some point of windows load causes this "visial darker" bug and clicking anything related to visual mode fixes it :)

All of them had SSd's changed and windows is installed cleanly (pro version).
Latest nvidia drivers and latest intel arc drivers.

Yea same thing happens with armoury crater as well (one of my LAZY FRIENDS is not using ghelper yet because hes on vacation, but ive just booted his laptop and its the same.

But i think you are right, it could be caused by intel arc drivers or asus system control interface.

**Reply by seerge** [reply]
@Ulvarin then if you prefer Visual Mode version of colors for your display the solution would be to forcibly apply even Default Native gamut every time app starts (or your laptop boots)

Just set Native gamut in G-Helper and then open `%appdata%\GHelper\log.txt`. Copy last call of `AsusSplendid.exe` with all the params from there and create a new task in windows Task Scheduler to run that on windows start (or user login).

**Reply by Ulvarin** [reply]
![image](https://github.com/user-attachments/assets/9b66d734-ae03-49ea-bc78-6494c6039a44)
this is the only one all of us are using :)
Ill wait with that fix for driver updates or something (especially form asus), im curious what is causing that on all units.
